src/components/Search.vue

```diff
@@ -89,10 +89,11 @@ v-row.align-center.girder-searchbar(no-gutters)
           v-for="r in quickResults",
           @click="$emit('select', r)",
           :key="r._id")
-        v-list-item-action.mr-2
-          v-icon {{ $vuetify.icons.values[r._modelType] }}
-        v-list-item-content
-          v-list-item-title {{ r.name || formatUsername(r) }}
+        slot(name="searchresult", v-bind="r")
```

Comment:
This feature looks good.
Could we rename the variable r to something like `record`?